app/controllers/resources_controller.rb

```diff
@@ -69,8 +75,10 @@ def find_resource_type
   end
 
   def find_integrations
+    parent_id = params[:parent_id] || (params.key?(:resource) && params[:resource][:parent_id])
+    is_a_dependent_resource = parent_id
```

Comment:
To ensure this is a boolean:

```suggestion
is_a_dependent_resource = parent_id.present?
```